renovate.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseBranches": ["main", "master","R2-2023","R1-2023", "R2-2023-Consortia","R1-2024","R1-2024-Consortia","R2-2024","R2-2024-Consortia"],
+  "baseBranches": ["main", "master","R2-2024","R2-2024-Consortia","R1-2025"],
```

Comment:
Maybe we should keep R1-2024 and R1-2024-Consortia?

Reply:
no problem, returned the value